src/ac_training_lab/cobot280pi/README.md

```diff
@@ -0,0 +1,44 @@
+# MyCobot 280 Pi
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/AccelerationConsortium/ac-training-lab/blob/cobot-usage-docs/src/ac_training_lab/cobot280pi/gradio-client-demo.ipynb)
+
+The general structure of the program is as follows;
+- The cobot runs a python server which listens for commands from HiveMQ, and runs the appropriate Python API functions from [pymycobot](https://github.com/elephantrobotics/pymycobot/blob/main/docs/MyCobot_280_en.md).
+- HuggingFace Spaces and the HuggingFace Python API (used in the Colab demo above) connect to HiveMQ and send commands to the appropriate topics, which the server is listening on.
+- If you own a Cobot, you can follow the instructions below to set it up. Once completed, you also have the option of using the `CobotController` class as defined in `client.py`.
+
+## Setup guide (if you own a cobot)
+1. Power on the Cobot280 Pi and connect the micro HDMI cable to a monitor.
+2. Connect the Cobot280 Pi to Wifi by clicking on the Wifi icon in the top right corner of the screen.
+3. Open a terminal and type to the following command to upgrade system packages
+```
+sudo apt-get update && sudo apt-get upgrade -y
+```
+4. Install the required system dependencies with the following command
+```
+sudo apt-get install git
+```
+5. Clone the repository containing the code for the server run by the Cobot.
```

Comment:
Are there few enough files that you could just point out which ones should be taken? (I.e., keeping it portable). How many files are needed?

At minimum, a comment on which files are most important would help.

Reply:
sounds good. added commands to move the required files out and delete the rest of the repo. does that work?

Reply:
Sure! That works

Reply:
so good to merge then?